packages/libs/src/sdk/types.ts

```diff
@@ -93,7 +96,13 @@ const DevAppSchema = z.object({
   /**
    * API secret, required for writes
    */
-  apiSecret: z.optional(z.string().min(1))
+  apiSecret: z.optional(z.string().min(1)),
+  /**
+   * Opt to use user (hedgehog) auth instead of
+   * by specifying app keys. User credentials will need to be
+   * supplied to sign requests.
+   */
+  userAuth: z.undefined()
```

Comment:
Why do we need `userAuth` in DevAppSchema and CustomAppSchema? Can we not use UserAppSchema for that exclusively

Reply:
I ran into some type errors... come to think of it I can' remember, so I'll take another swing at this